css/index-about.css

```diff
@@ -0,0 +1,175 @@
+/* http://meyerweb.com/eric/tools/css/reset/
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed,
+figure, figcaption, footer, header, hgroup,
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure,
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Roboto', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-size: 18px;
+    margin-bottom: 15px;
+    font-family: 'Rubik', sans-serif;
+}
+
+p {
+    line-height: 1.4;
+}
+
+
+/* Container  */
+.container {
+    width: 800px;
+    margin: 0 auto;
+}
+
+/* Nav  */
+ header nav {
+	display: flex;
+	justify-content: space-between;
+	align-items: baseline;
+	padding: 50px 0;
+  	width: 95%;
+}
+
+header  nav a {
+	text-decoration: none;
+	color: black;
+}
+
+header nav a.transform:hover {
+	animation-name: fadeIn;
+	animation-duration: 1.5s;
+	animation-iteration-count: 2;
+	animation-direction: alternate;
+	border-radius: 5px;
+	/* padding: 5px 0; */
+	width: 75px;
+	text-align: center;
+}
+
+@keyframes fadeIn {
+	from {
+		background: inherit;
+		box-shadow: none;
+	}
+
+	to {
+		background: lightgray;
+		box-shadow:0 0 0 5px #ccc
+	}
+}
+/* Header styles  */
+
+header {
+  border-bottom: 1px dotted black;
+  margin-bottom: 50px;
+}
+.header-img {
+	margin-bottom: 50px;
+}
+
+/* Info Area */
+
+.about-page {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  margin-bottom: 40px;
+}
+
+.about-page .info-box {
+  width: 46%;
+  border-bottom: 1px dotted black;
+  margin-bottom: 50px;
+  padding-bottom: 20px;
+}
+
+.about-page .bottom-box {
+  width: 95%;
+}
+
+.about-page .info-box img {
+  margin-bottom: 15px;
+}
+
+.about-page .info-box button {
+  margin-top: 15px;
+  background: white;
+  border-radius: 10px;
+  border: 1.7px solid black;
+  padding: 5px;
+  width: 35%;
+  cursor: pointer;
+}
+
```

Comment:
you may be able to achieve the same results and not be as specific here, i.e. instead of `.about-page .info-box button`,  perhaps `.info-box button` could have achieved the same results and been less writing